.github/workflows/build.yml

```diff
@@ -38,19 +38,18 @@ jobs:
       - name: Setup java
         uses: actions/setup-java@v3
         with:
-          distribution: 'adopt'
+          distribution: 'corretto'
           java-version: '11'
           cache: 'sbt'
- 
+
       - name: Build
         run: |
           ./scripts/build.sh
-          
+
       - name: Upload to Riff-Raff
         uses: guardian/actions-riff-raff@v2
         with:
-          app: cv-redact-tool
```

Comment:
`app` is redundant as it is ignored when `projectName` is set - https://github.com/guardian/actions-riff-raff#available-inputs.